### PR TITLE
Emagged Borg Sound Effect

### PR DESCRIPTION
**Pull request by ps3moira**
## About the PR

- Added a sound effect when Borgs are Emagged

## Media

**Changelog**
:cl:
- add: Emagged Borg Sound Effect

### CODE REVIEW

File: Resources/Audio/Ambience/Antag/attributions.yml [hunk 1]
```diff
@@ -9,4 +9,8 @@
 - files: ["zombie_start.ogg"]
   license: "CC-BY-NC-SA-3.0"
   copyright: "Made by @MIXnikita#1474 (Discord) for SS14"
-  source: "https://github.com/SerbiaStrong-220/space-station-14/commit/f33ce80db9d8ac962ec36eb5ae61afa8c0035d1a"
\ No newline at end of file
+  source: "https://github.com/SerbiaStrong-220/space-station-14/commit/f33ce80db9d8ac962ec36eb5ae61afa8c0035d1a"
```
Comment (t1m1yep) [inline Resources/Audio/Ambience/Antag/attributions.yml:12]: SS220 server have license with hosting restriction

Comment (ps3moira) [reply Resources/Audio/Ambience/Antag/attributions.yml:12]: What does this mean?

Comment (metalgearsloth) [reply Resources/Audio/Ambience/Antag/attributions.yml:12]: As this change isn't from this PR please ignore this.

Comment (ps3moira) [reply Resources/Audio/Ambience/Antag/attributions.yml:12]: > As this change isn't from this PR please ignore this.

Ah, I think I understand.

**Comment by deltanedas** [comment]
cl is worded wrong

**Comment by ps3moira** [comment]
> cl is worded wrong

How would it be worded?

**Comment by deltanedas** [comment]
> How would it be worded?

Emagging borgs now plays a sound effect.

**Pull request closed without merging**